### PR TITLE
Add bottom navigation tab

### PR DESCRIPTION
**Pull request by psakurai**

### CODE REVIEW

**Review by mangkoran — COMMENTED** [review]
For next commit, make sure to put the exact same name as the package name, e.g., `build: add @react-navigation/bottom-tabs`

**Review by mangkoran — CHANGES_REQUESTED** [review]
For `tab.js`, as it is a `component`, you may create new directory and put `tab.js` inside it.

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     }
   },
   "dependencies": {
+    "@react-navigation/bottom-tabs": "^6.0.9",
```
Comment (mangkoran) [inline package.json:19]: This is the only package you committed.

Comment (psakurai) [reply package.json:19]: I haven't push the other 2 packages.

Comment (mangkoran) [reply package.json:19]: Make sure to follow https://github.com/t-analog/crowdfinder-client/pull/39#pullrequestreview-817429076.

**Review by mangkoran — APPROVED** [review]

**Review by mangkoran — CHANGES_REQUESTED** [review]
Because since [this commit](https://github.com/t-analog/crowdfinder-client/pull/39/commits/de44b425e3678a1eeecc870f91c9b13ba5227c7d) you actually add another functionality unrelated to PR title.

**Review by mangkoran — APPROVED** [review]

**Pull request closed without merging**